nova/virt/vmwareapi/cluster_util.py

```diff
@@ -429,3 +430,26 @@ def update_cluster_das_vm_override(session, cluster, vm_ref, operation='add',
     config_spec.dasVmConfigSpec = [das_vm_spec]
 
     reconfigure_cluster(session, cluster, config_spec)
+
+
+def get_cluster_median_cpu_mhz(session, cluster) -> int:
+    """Get the median CPU frequency for all hosts in a cluster.
```

Comment:
I don't think `cluster_util` is used for this kind of functionality (which calculates something about hosts). Instead, I'd put it where we have the failover-host detection and such.

Btw. why do we have to fetch these things anew? Could we not take the `VCState` information for this? It should™ contain all the hosts and their cpu count. If we now also need the `hz`, maybe it would be best to add it there.

Reply:
obsoleted